src/protocol/libp2p/identify.rs

```diff
@@ -265,15 +281,13 @@ impl Identify {
             }
         };
 
+        let listen_addrs =
```

Comment:
To comply with libp2p implementation, we should include both public addresses and listen addresses here.